src/main/kotlin/com_github_leetcode/children/Node.kt

```diff
@@ -0,0 +1,27 @@
+package com_github_leetcode.children
```

Comment:
Please reuse nighbous Node

```
public class Node {
    public int val;
    public List<Node> neighbors;
```

Reply:
this Node is different from with neighbors

Reply:
I know. I leetcode-in-java we used it for solutions.